v2/datastream-to-spanner/terraform/samples/postgresql-end-to-end/terraform.tfvars

```diff
@@ -0,0 +1,93 @@
+# Common Parameters
+common_params = {
+  project                         = "<YOUR_PROJECT_ID>"      # Replace with your GCP project ID
+  host_project                    = "<YOUR_HOST_PROJECT_ID>" # If you are using a shared VPC
+  region                          = "<YOUR_GCP_REGION>"      # Replace with your desired GCP region
+  migration_id                    = "<YOUR_MIGRATION_ID>"    # Will be used as a prefix for all resources, auto-generated if not specified
+  add_policies_to_service_account = "<TRUE/FALSE>"           # This will decide if roles will be attached to service accounts or not.
+}
+
+# Datastream Parameters
+datastream_params = {
+  source_connection_profile_id  = "source-postgresql" # Or provide a custom ID
+  target_connection_profile_id  = "target-gcs"        # Or provide a custom ID
+  target_gcs_bucket_name        = "live-migration"    # Or provide a custom bucket name
+  pubsub_topic_name             = "live-migration"    # Or provide a custom topic name
+  stream_id                     = "postgresql-stream" # Or provide a custom stream ID
+  max_concurrent_cdc_tasks      = 50                  # Adjust as needed
+  max_concurrent_backfill_tasks = 50                  # Adjust as needed
+  postgresql_host               = "<YOUR_POSTGRESQL_HOST_IP_ADDRESS>"
+  # Use the Public IP if using IP allowlisting and Private IP if using
+  # private connectivity.
+  postgresql_username         = "<YOUR_POSTGRESQL_USERNAME>"
+  postgresql_password         = "<YOUR_POSTGRESQL_PASSWORD>"
+  postgresql_publication      = "<YOUR_POSTGRESQL_PUBLICATION>"
+  postgresql_replication_slot = "<YOUR_POSTGRESQL_REPLICATION_SLOT>"
+  postgresql_port             = 3306
+  postgresql_database = {
+    database = "<YOUR_DATABASE_NAME>"
+    tables   = [] # List specific tables to replicate (optional)
+  }
```

Comment:
This will need to evolve to support PG's schema concept. There should be database -> schema -> tables in here.

Reply:
Agreed. Lets discuss on how to do that. I can raise a different PR accordingly.

Reply:
We can do this as part of this PR itself, let's discuss over chat.

Reply:
Done